File: py/1117.py
scores = []
while True:
    n = float(input())
    
    if n >= 0 and n <= 10:
        scores.append(n)
    
    else:
        print('nota invalida')
        
    if len(scores) == 2:
        print(f'media = {(sum(scores) / len(scores)):.2f}')
        break

File: cpp/1117.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int count = 0;
    float score, sum = 0;

    while (count < 2) {
        cin >> score;

        if (0 <= score && score <= 10) {
            sum += score;
            count++;
        } else {
            cout << "nota invalida\n";
        }
    }

    printf("media = %.2f\n", sum / 2);
    return 0;
}